README.md

```diff
@@ -9,16 +9,33 @@ Bowling Challenge
 
 ## The Task
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. AN ACTUAL USER INTERFACE IS OPTIONAL**
 
-Count and sum the scores of a bowling game for one player (in JavaScript).
+Count and sum the scores of a bowling game for one player (in JavaScript). 
 
 A bowling game consists of 10 frames in which the player tries to knock down the 10 pins. In every frame the player can roll one or two times. The actual number depends on strikes and spares. The score of a frame is the number of knocked down pins plus bonuses for strikes and spares. After every frame the 10 pins are reset.
 
+Start by looking in detail at the rules and the example of scoring for a complete game given below.
+
+An example of how your code might be used could be:
+
+```javaScript
+let scorecard = new Scorecard()
+scorecard.addRoll(5)
+scorecard.calculateScore() // returns 5
+scorecard.addFrame([3, 5]) // returns "Please finish the current frame [5, ?]"
+scorecard.addRoll(2)
+scorecard.addFrame([3, 5])
+scorecard.calculateScore() // returns 15
+scorecard.showScore() // [[5, 2], [3, 5]]
```

Comment:
I would tend to remove this one, as it would tend to expose the internal state of the rolls data — the main behaviour we want the class / program to have is to calculate the final score, so I would say the `calculateScore()` method is the main one "returning" something. What do you think?